manifests/cf-manifest/manifest/000-base-cf-deployment.yml

```diff
@@ -33,7 +33,7 @@ releases:
 stemcells:
   - alias: default
     name: bosh-aws-xen-hvm-ubuntu-trusty-go_agent
-    version: 3232.11
+    version: 3262.2
```

Comment:
Could you omit this change from the PR?

We were hoping to change the stemcell independently in #370 and it will cause a merge conflict if it's also changed here. We have found that, so far, it's OK to change the stemcell at a difference cadence from cf-release.

Out of interest, could you describe why you chose the `3262` series of stemcell instead of `3232`?